File: RetiocusWebUtils/UtilsEntities/Message.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace UtilsEntities
{
    public class Message
    {
        public string remitente { get; }
        public string cuerpo { get; }
        public long timeInMillis { get; }

        public Message(string remitente, string cuerpo, long timeInMillis)
        {
            this.remitente = remitente;
            this.cuerpo = cuerpo;
            this.timeInMillis = timeInMillis;
        }
    }
}

File: RetiocusApp/app/src/main/java/com/jruiz/retiocusapp/signalr/models/Message.java
package com.jruiz.retiocusapp.signalr.models;

import java.time.LocalDateTime;
import java.time.OffsetDateTime;
import java.util.Calendar;
import java.util.Date;

public class Message {
    private String remitente, cuerpo;
    private long timeInMillis;

    public Message(String remitente, String cuerpo) {
        this.remitente = remitente;
        this.cuerpo = cuerpo;
        timeInMillis=new Date().getTime();
    }

    public String getRemitente() {
        return remitente;
    }

    public void setRemitente(String remitente) {
        this.remitente = remitente;
    }

    public String getCuerpo() {
        return cuerpo;
    }

    public void setCuerpo(String cuerpo) {
        this.cuerpo = cuerpo;
    }

    public long getFechaEnvio() {
        return timeInMillis;
    }

    public void setFechaEnvio(long timeInMillis) {
        this.timeInMillis = timeInMillis;
    }
}
